runtime/src/tests/work-item/work-item-list/page-objects/work-item-detail.page.js

```diff
@@ -414,6 +414,8 @@ class WorkItemDetailPage {
     return element(by.id('comment_time_'+index)).getText();
   }
   commentEdit(index) {
+    browser.actions().mouseMove(element(by.css('#comment_body_' + index))).perform();
+    browser.wait(until.elementToBeClickable(element(by.css('#comment_body_' + index + ' .edit-icon'))), constants.WAIT, 'Failed to find comment edit button');
```

Comment:
This line crosses the the maximum number of column standard, please break it in 2-3 lines.